app/src/component/customRewardCard/view.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import React from 'react';
+import { ActivityIndicator, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import Colors from '../../styles/colors';
```

Comment:
Have you looked at using aliases for these through babel?

This would be ideal since we can use it across all apps:
https://www.npmjs.com/package/babel-plugin-webpack-aliases

And here's an alternative that isn't opinionated towards how Webpack does things:
https://www.npmjs.com/package/babel-plugin-module-resolver

Reply:
I'll look into this.